app/src/main/java/org/mozilla/fenix/browser/WebAuthnFeature.kt

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.browser
+
+import android.app.Activity
+import android.content.Intent
+import android.content.IntentSender
+import mozilla.components.concept.engine.Engine
+import mozilla.components.concept.engine.activity.ActivityDelegate
+import mozilla.components.support.base.feature.ActivityResultHandler
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+import mozilla.components.support.base.log.logger.Logger
+
+/**
+ * This implementation of the WebAuthnFeature is only for testing in a nightly signed build.
+ *
+ * ⚠️ IT SHOULD NOT GO TO RELEASE.
```

Comment:
nit: maybe link to the feature flag here, as the code will go to release, it just won't be enabled :).

Reply:
Good point, this comment will probably cause confusion otherwise.